common/src/main/java/io/netty/util/ResourceLeakDetectorFactory.java

```diff
@@ -56,23 +56,22 @@ public static void setResourceLeakDetectorFactory(ResourceLeakDetectorFactory fa
      * Returns a new instance of a {@link ResourceLeakDetector} with the given resource class.
      *
      * @param resource the resource class used to initialize the {@link ResourceLeakDetector}
-     * @param <T> the type of the resource class
+     * @param <T>      the type of the resource class
```

Comment:
Can you please revert any change which is just reformatting ?

Reply:
ok,i will revert any code format

Reply:
Sorry, forgive me for not understanding the formatting style of netty code, if there is pr in the future, I will pay attention to this

Reply:
@skyguard1 don't worry... thanks for the PR :)